File: math.py
#jueo de matematicas, numbers y simbolos aleatorios

import random
print("Hecho por Alex_dev")
print(" ")
def jugar():
    print("Escoje una dificuldad: (Fácil 1-10)  (Medio 1-50)  (Díficil 1-100) (Extremo 100-500)")
    print("Las operaciones seran: +, -, *")
    dificultad = input("Dificultad: ").capitalize()
    if dificultad == "Facil":
        numbers = list(range(1, 11))
    elif dificultad == "Medio":
        numbers = list(range(1, 51))
    elif dificultad == "Dificil":
        numbers = list(range(1, 101))
    elif dificultad == "Extremo":
        numbers = list(range(100, 501))
    num1 = random.choice(numbers)
    num2 = random.choice(numbers)
    simbolos = [ "+", "-", "*", ]
    simbolo = random.choice(simbolos)

    op = f"{int(num1)} {simbolo} {int(num2)}"
    respuesta = eval(op)
    print(" ")
    print("Resuelve lo siguiente: ",op)
    user = int(input("Respuesta: "))

    if user == respuesta:
        print("¡Que insano eres mi crack!, Correcto")
    else:
        print("¡Incorrecto mi crack!")
    print(" ") 

jugar_de_nuevo = "si"
while jugar_de_nuevo.lower() == "si":
    jugar()
    print("¿Quieres jugar otra vez? (si/no)")
    jugar_de_nuevo = input().lower()

File: matematicas/math.js
function elegirAleatorio(lista) {
        let indice = Math.floor(Math.random() * lista.length);
        return lista[indice];
}

const readline = require('readline');

const rl = readline.createInterface({
        input: process.stdin,
        output: process.stdout
});

let numeros = [];
let simbolos = ["+", "-", "*"];

console.log("(facil)  (medio)  (dificil)");
rl.question("Elige una dificultad: " , function(dificultad){
        if (dificultad == "facil") {
                numeros = [1,2,3,4,5,6,7,8,9,10];
        }
        else if (dificultad == "medio") {
                numeros = [1,2,3,4,5,6,7,8,9,10,11,12,13,14,15,16,17,18,19,20];
        }
        else if (dificultad == "dificil") {
                numeros = [11,12,13,14,15,16,17,18,19,20,21,22,23,24,25,26,27,28,29,30,31,32,33,34,35,36,37,38,39,40,41,42,43,44,45,46,47,48,49,50];
        }

        let num1 = elegirAleatorio(numeros);
        let num2 = elegirAleatorio(numeros);
        let simbolo = elegirAleatorio(simbolos);
        let operacion = num1 + simbolo + num2;
        let respuesta = eval(operacion);
        console.log(operacion);

        rl.question("¿Cual es el resultado? " , function(answer){
                if (answer == respuesta){
                        console.log("Correcto");
                }
                else {
                        console.log("Incorrecto");
                }
                rl.close();
        });
});